lib/ws_servers/api/send_authenticated.js

```diff
@@ -49,7 +49,7 @@ module.exports = async (ws, db, d) => {
     switch (exID) {
       case 'bitfinex': {
         ws.aoc.openHost('bitfinex', key, secret)
-        ws.clients.bitfinex = await openAuthBitfinexConnection(ws, key, secret, db, d)
+        ws.clients.bitfinex = await openAuthBitfinexConnection(ws, key, secret, db, d, opts)
```

Comment:
would it be possible to pass object literal as `openAuthBitfinexConnection` argument instead of having so many? Having object has just too many advantages over multiple arguments